.github/workflows/pr-comment.yaml

```
@@ -10,4 +10,4 @@ jobs:
     if: ${{ github.event.label.name == 'queued' }} || contains(github.event.comment.body, 're')
     runs-on: self-hosted
     steps:
-      - run: "echo Found it!"
+      - run: "echo Found it!
```

actionlint (1.7.3)

could not parse as YAML: yaml: line 13: found unexpected end of stream

(syntax-check)